src/features/comments.ts

```diff
@@ -0,0 +1,108 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+export const loadComments = createAsyncThunk(
+  'comments/loadComments',
+  async (postId: number) => {
+    const comments = await getPostComments(postId);
+
+    return comments;
+  },
+);
+
+export const createNewComment = createAsyncThunk(
+  'comments/createNewComment',
+  async (data: Omit<Comment, 'id'>) => {
+    const comments = await createComment(data);
+
+    return comments;
+  },
+);
+
+export const deleteCommentFromServer = createAsyncThunk(
+  'comments/delete',
+  async (id: number) => {
+    await deleteComment(id);
+
+    return id;
+  },
+);
+
+export type CommentsState = {
+  comments: Comment[];
+  loading: string;
+  error: string;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loading: '',
+  error: '',
+};
+
+export const commentsSlice = createSlice({
+  name: 'comments',
+  initialState,
+  reducers: {},
+  extraReducers: builder => {
+    builder
+      .addCase(loadComments.pending, state => {
+        /* eslint-disable-next-line no-param-reassign */
```

Comment:
Add this eslint rule only to the beginning of the file and remove it everywhere else